pkgs/tools/networking/boringtun/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "boringtun";
+  version = "0.2.0";
```

Comment:
The Github page does not show any release yet and states that the software is still under security review. I would suggest to use `20190407` as version number to indicate that it is an unstable version.

Reply:
Thanks for the review!
Let's use `20190407` as version number to indicate that it is an unstable version.